apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QMapCompactDisabledIT.java

```diff
@@ -0,0 +1,126 @@
+package io.stargate.sgv2.it;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.quarkus.test.junit.QuarkusTestProfile;
+import io.quarkus.test.junit.TestProfile;
+import io.stargate.sgv2.common.testresource.StargateTestResource;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+@QuarkusIntegrationTest
+@QuarkusTestResource(StargateTestResource.class)
+@TestProfile(RestApiV2QMapCompactDisabledIT.Profile.class)
+public class RestApiV2QMapCompactDisabledIT extends RestApiV2QIntegrationTestBase
+    implements RestApiV2QMapTests {
+  private static final boolean SERVER_FLAG = false;
```

Comment:
Could use Javadoc comment -- and maybe rename if it's for compact-map-data setting.

Reply:
Changed the name and updated the JavaDoc as well